spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1317,14 +1319,19 @@ The type conversions Spring Cloud Stream provides out of the box are summarized
 
 |===
 
-[[NOTE]]
+[NOTE]
+====
 Conversion applies to payloads that require type conversion.
 For example, if a module produces an XML string with outputType=application/json, the payload will not be converted from XML to JSON.
-This is because the payload at the module's output channel is already a String so no conversion will be applied at runtime.
+This is because the payload at the module's outbound channel is already a String so no conversion will be applied at runtime.
```

Comment:
Not entirely clear why we needed to change 'input'->'inbound' and 'output'->'outbound' ? Both are valid choices, but we use `@Input` and `@Output`.